etna/transforms/missing_values/imputation.py

```diff
@@ -86,7 +88,7 @@ def __init__(
         self.window = window
         self.seasonality = seasonality
         self.default_value = default_value
-        self.fill_value: Optional[int] = None
+        self.fill_value: Optional[float] = None
```

Comment:
Let's place the constant_value initialization before this line

Reply:
https://github.com/tinkoff-ai/etna/pull/819/files#diff-345da9722fb55e4c74ad4e4c73fdc9eca093844b5705985c38bd4b5272e4c774R87
?